src/app/components/navigation/TrackedRoute.tsx

```diff
@@ -52,11 +53,16 @@ const WrapperComponent = function ({ component: Component, trackingOptions, ...p
   );
 };
 
+export const isGoogleBot = function (userAgent?: string): boolean {
+  const googleBotUserAgents: string = "compatible; Googlebot/2.1; +http://www.google.com/bot.html";
+  return userAgent !== undefined && userAgent.includes(googleBotUserAgents);
```

Comment:
Sonarcloud suggests optional chaining here:

`return userAgent?.includes(googleBotUserAgents)`

Reply:
I noticed that and already tried that method. 
However it is not working that way. There are no errors really, but as a user you can bypass login. 
That is why I kept it like that.

Reply:
This way is working:  

` return userAgent?.includes(googleBotUserAgents) ?? false;`

It was missing the last part